helm/reana/README.md

```diff
@@ -55,6 +55,7 @@ This Helm automatically prefixes all names using the release name to avoid colli
 | `node_label_runtimebatch`                                | Define the label which identifies the nodes where the runtime workflow pods should run, e.g. `reana.io/system=runtimebatch`. By default runtime workflow pods can be scheduled to any available node in the cluster. | None |
 | `node_label_runtimejobs`                                 | Define the label which identifies the nodes where the runtime job pods should run, e.g. `reana.io/system=runtimejobs`. By default runtime job pods can be scheduled to any available node in the cluster. | None |
 | `node_label_runtimesessions`                             | Define the label which identifies the nodes where the runtime session pods should run, e.g. `reana.io/system=runtimesessions`. By default runtime session pods can be scheduled to any available node in the cluster. | None |
+| `workflow_scheduling_policy`                             | Define workflow scheduling strategy. Options are "fifo" for first-in-first-out strategy regardless of users and "balanced" for multi-user-aware scheduling strategy. | "balanced" |
```

Comment:
Please move down so that the order is alphabetical... However, since we usually use `components.reana_stuff.variable` nomenclature, we should perhaps call it `components.reana_server,workflow_scheduling_policy`?